pyproject.toml

```
@@ -1,3 +1,41 @@
+[project]
+name = "spidev"
+description = "Python bindings for Linux SPI access through spidev"
+dynamic = ["readme"]
+version = "3.8"
```

Instead of duplicating the version here, we can still parse it from the C file with a little overhead. I would suggest to do:

```suggestion
dynamic = ["readme", "version"]
```

And then keep the `setup.py` script with the following content to parse the version string from the C file and pass it to the setup routine:

```python
#!/usr/bin/env python

from setuptools import setup

version = "0.0"

lines = [x for x in open("spidev_module.c").read().split("\n") if "#define" in x and "_VERSION_" in x and "\"" in x]

if len(lines) > 0:
    version = lines[0].split("\"")[1]
else:
    raise Exception("Unable to find _VERSION_ in spidev_module.c")

setup(
    version = version,
)
```